README.md

```diff
@@ -80,6 +82,10 @@ Here are few points before you start contributing:
 * Database connection is at `internal/db`, cloud connections are in 'internal/clouds'.
 * Do not introduce `utils` or `tools` common packages, they tend to grow.
 * Keep the line of sight (happy code path).
+* PostgreSQL version we currently use in production is v14+ so take advantage of all modern SQL features.
+* Use `BIGINT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY` for primary keys.
+* Use `TEXT` for string columns, do not apply any limits in the DB: https://wiki.postgresql.org/wiki/Don%27t_Do_This#Text_storage
+* Keep security in mind: https://github.com/RedHatInsights/secure-coding-checklist
```

Comment:
Should these be rationalized by ADR? I'd be ok with a quickie directly to git, summarizing your thoughts, how you got to these conclusions, I guess not much discussion needed, but having rationale backing these decisions would be useful ^_^

Reply:
I've taken the PK and TEXT decisions from https://github.com/RedHatInsights/vuln4shift-backend/blob/master/dbadmin/migrations/001_init_schema.up.sql and the checklist is from https://github.com/RedHatInsights/vuln4shift-backend/blob/master/pull_request_template.md as well. I can remove whatever you like from this PR, no big deal, I just thought its useful.

Reply:
No need, I'll add the ADR into git myself, don't get me wrong I'm all up for it, and I don't mind slightly bigger PR at the beggining, but we will for sure wonder in the future why it is there, so I'd like to have that written down somewhere, but I'm happy to do that! :)

Reply:
Let me know if you want me to remove this from the PR.